WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductRowViewModel.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import Yosemite
+
+/// View model for `ProductRow`.
+///
+final class ProductRowViewModel: ObservableObject, Identifiable {
+    /// Product ID
+    /// Required by SwiftUI as a unique identifier
+    ///
+    let id: Int64
+
+    private let currencyFormatter: CurrencyFormatter
+
+    /// Whether the product quantity can be changed.
+    /// Controls whether the stepper is rendered.
+    ///
+    let canChangeQuantity: Bool
+
+    /// Product to display
+    ///
+    let product: Product
+
+    /// Label showing product name
+    ///
+    var nameLabel: String {
+        product.name
+    }
+
+    /// Label showing product stock status and price.
+    ///
+    var stockAndPriceLabel: String {
```

Comment:
Could this be lazy so we don't have to recompute it, every time the view gets re-rendered?